esp-hal/src/sdio.rs

```diff
@@ -0,0 +1,586 @@
+//! # Secure Digital I/O - Slave Mode
```

Comment:
This should be sdio/slave.rs

Reply:
Right, I think I copied this doc string from somewhere else. I'll move the current top-level stuff into a separate module.

Reply:
SDIO is always a slave driver.

- `src/sdio.rs`
- `src/sdmmc.rs`

Reply:
Oki disregard me, I'm rather confused about the different names in this area, or even how the whole thing is supposed to work. But if it's always slave mode, then there's no reason to call out that it's slave mode.